Apps/APN-UIKit/APN UIKit/View/LoginViewController.swift

```diff
@@ -111,6 +115,6 @@ class LoginViewController: BaseViewController {
     }
 
     func userDetailsValid() -> Bool {
-        return !emailTextField.isTextTrimEmpty
+        !emailTextField.isTextTrimEmpty
```

Comment:
TIL: Swift allows skipping the return keyword when there is only one expression in function. Interesting 🤓